dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/zk/ZKMasterClient.java

```diff
@@ -272,9 +286,8 @@ private boolean checkTaskAfterWorkerStart(TaskInstance taskInstance) {
 
 		if(workerServerStartDate != null){
 			return taskInstance.getStartTime().after(workerServerStartDate);
-		}else{
-			return false;
-		}
+        }
+		return false;
```

Comment:
Indent problem

Reply:
hi, thanks for your review , I have resolved the code style, please take a look.